lib/const.rb

```diff
@@ -76,17 +76,21 @@
 CREW_META_PATH        ||= File.join(CREW_CONFIG_PATH, 'meta')
 CREW_BREW_DIR         ||= File.join(CREW_PREFIX, 'tmp/crew')
 CREW_DEST_DIR         ||= File.join(CREW_BREW_DIR, 'dest')
-CREW_WINE_PREFIX      ||= File.join(CREW_LIB_PREFIX, 'wine')
 CREW_DEST_PREFIX      ||= File.join(CREW_DEST_DIR, CREW_PREFIX)
 CREW_DEST_LIB_PREFIX  ||= File.join(CREW_DEST_DIR, CREW_LIB_PREFIX)
-CREW_DEST_WINE_PREFIX ||= File.join(CREW_DEST_PREFIX, CREW_WINE_PREFIX)
 CREW_DEST_MAN_PREFIX  ||= File.join(CREW_DEST_DIR, CREW_MAN_PREFIX)
+CREW_WINE_PREFIX      ||= File.join(CREW_LIB_PREFIX, 'wine')
+CREW_DEST_WINE_PREFIX ||= File.join(CREW_DEST_PREFIX, CREW_WINE_PREFIX)
 
 # Local constants for contributors.
 CREW_LOCAL_REPO_ROOT ||= `git rev-parse --show-toplevel 2> /dev/null`.chomp
 CREW_LOCAL_BUILD_DIR ||= "#{CREW_LOCAL_REPO_ROOT}/release/#{ARCH}"
 CREW_GITLAB_PKG_REPO ||= 'https://gitlab.com/api/v4/projects/26210301/packages'
 
+# Glibc related constants
+CREW_GLIBC_PREFIX      ||= File.join(CREW_LIB_PREFIX, 'opt/glibc-libs')
```

Comment:
This needs to be : 
```ruby
CREW_GLIBC_PREFIX ||= File.join(CREW_PREFIX, 'opt/glibc-libs')
```

Reply:
You could also take out the `opt/`.

Reply:
Fixed

Reply:
That would require rebuilds of glibc_standalone and also trip the warnings for FHS compatibility.